README.md

```diff
@@ -0,0 +1,106 @@
+# Elevator_Simulator
+
+Elevator_Simulator is a software that simulates Elevator in a commercial building.
+
+
+# Authors
+
+- Hariprasad Munusamy - HariprasadMunusamy@cmail.carleton.ca (101132017) - Carleton University
```

Comment:
is it nessary to put out emails and ids?  Its information we shouldnt really share out in public.